docs/source/share.rst

```diff
@@ -0,0 +1,562 @@
+Share
+======
+
+At Hugging Face, we are on a mission to democratize NLP and we believe in the value of open source. That's why we designed 🤗 Datasets so that anyone can share a dataset with the greater NLP community. There are currently over 900 datasets in over 100 languages, and the Hugging Face team always welcomes new submissions!
+
+This guide will show you how to add a dataset that can be easily accessed by anyone.
+
+There are two options to add a new dataset:
+
+- directly upload it on the Hub as a community provided dataset.
+- add it as a canonical dataset by opening a pull-request on the `GitHub repository for 🤗 Datasets <https://github.com/huggingface/datasets>`__,
+
+Community vs. Canonical
+-----------------------
+
+Both options offer the same features such as:
+
+- dataset versioning
+- commit history and diffs
+- metadata for discoverability
+- dataset cards for documentation, licensing, limitations, etc.
+
+The main differences between the two are highlighted in the table below:
+
+.. list-table::
+    :header-rows: 1
+
+    * - Community datasets
+      - Canonical datasets
+    * - Faster to share, no review process.
+      - Slower to add, needs to be reviewed.
+    * - Data files can be stored on the Hub.
+      - Data files are typically retrieved from the original URLs.
+    * - Identified by a user or organization namespace like **thomwolf/my_dataset** or **huggingface/our_dataset**.
+      - Identified by a root namepsace. Need to select a short name that is available.
+    * - Requires data files and/or a dataset script.
+      - Always requires a dataset script.
+    * - Flagged as **unsafe** because the dataset contains executable code.
+      - Flagged as **safe** because the dataset has been reviewed.
+
+For community datasets, you can either have your dataset in a supported format (csv/jsonl/json/parquet/txt), or use a dataset script to define how to load your data.
+If your dataset is in a supported format, you're all set ! In this case you can directly skip the dataset script guide and go the Dataset card and Upload sections.
+Otherwise, you need a dataset script. It simply is a python script and its role is to define the dataset splits, the feature types and how to download and process the data.
```

Comment:
Maybe we can have a section that shows how to share a community datasets without dataset script with more details. I think we want to advertise more this new feature

Reply:
I added more details and had to separate the Share page